scripts/build.sh

```diff
@@ -11,13 +11,24 @@ BASE_ARGS="\
     -DENABLE_JIMPLE_FRONTEND=On \
     -DCMAKE_INSTALL_PREFIX:PATH=$PWD/release \
 "
+# Must disable old frontend to enable goto contractor. github issue #1110
+# https://github.com/esbmc/esbmc/issues/1110.
```

Comment:
Once should be enough, we can remove this comment here.

Reply:
Removed.